src/main/java/org/arquillian/container/chameleon/spi/model/Target.java

```diff
@@ -5,7 +5,7 @@
 public class Target {
 
     public static enum Type {
-        Remote, Managed, Embedded, Default;
+        Remote, Managed, Embedded, Domain, Default;
```

Comment:
hmm.. This would hard code domain to be Managed(or whatever Type: domain is configured to). Domain can technically be both Remote and Managed.

Wondering if this would be better as a 'server', e.g. WildFly Domain:10.0.0.Final:managed